src/App.js

```diff
@@ -0,0 +1,25 @@
+import React, { Component } from 'react';
+import Welcome from './components/calculator';
+
+class App extends Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+
+    };
+  }
+
+  render() {
+    return (
+      <div className="App">
+
+        <h1>MY CALCULATOR</h1>
+        <Welcome />
```

Comment:
- [ ] After importing `Calculator`, here you will use it as `Calculator` instead of ~`Welcome`~
```suggestion
        <Calculator />
```